CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 1.1.3
```

Comment:
will need to target at least `1.2.0` (maybe even `1.3.0` as I think `1.2.0` will be released soon)